packages/invoices/src/OutboundShipment/DetailView/modals/hooks/useNextItem.test.tsx

```diff
@@ -137,10 +137,10 @@ describe('useNextItem', () => {
   });
 
   it('eventually equals an object where the next item is equal to the next item in sorted order.', () => {
-    const invoice = getInvoice();
-    invoice.lines.nodes = getLines();
-    const handler = graphql.query('invoice', (_, res, ctx) => {
-      return res(ctx.data({ invoice }));
+    const invoiceByNumber = getInvoice();
```

Comment:
test updates from using `invoiceByNumber` rather than `invoice` query in a older PR